wmn-data.json

```diff
@@ -4510,6 +4510,16 @@
         "known" : ["claudii", "kowalski"],
         "cat" : "social"
        },
+       {
+        "name" : "Quizlet",
+        "uri_check" : "https://quizlet.com/user/{account}/sets",
+        "e_code" : 200,
+        "e_string" : "Flashcard sets",

```

Comment:
This is too generic an indicator as it appears in multiple places in the response. Please consider changing to something like `f1qxvjkt` with is the class when there is a user.

Reply:
Yep, my mistake. Using `f1qxvjkt` doesn't seem to be a good approach, `AssemblyShimmer` seems to fit.